### ISSUE TITLE
Min

### ISSUE THREAD

**Issue by sreikes7** [Richard2926/math_lab#1]
Hi,

Thank you so much for making this! I was just practicing this morning with "Spring 2022 Practice". I entered in 2 fake students, and I was getting an error when "generating schedule". I am wondering if the min needs to be set to zero incase not every time slot is selected?

**Comment by Richard2926** [comment]
You are right Ms. Reikes, if there is only one student who needs 4 1/2 hour time slots and we have 5 1/2 hour time slots, the schedule can't be generated since we set the min to 1. Please reopen if you want the schedule to set min to 0 if not enough students are there. I thought that since you'll always have around 20 students, this will not be an issue.

**Comment by sreikes7** [comment]
Probably won't be an issue, but set to 0 just incase!

**Comment by Richard2926** [comment]
Okay! Just to clarify, would you like it to be set to 0 even if there are enough students to fill or just when there aren't enough?
